src/mono/mono/metadata/icall.c

```diff
@@ -5814,7 +5814,7 @@ ves_icall_AssemblyExtensions_ApplyUpdate (MonoAssembly *assm,
 	g_assert (image_base);
 	// TODO: use dpdb_bytes
 
-	mono_image_load_enc_delta (mono_get_root_domain (), image_base, dmeta_bytes, dmeta_len, dil_bytes, dil_len, error);
+	mono_image_load_enc_delta (image_base, dmeta_bytes, dmeta_len, dil_bytes, dil_len, error);
```

Comment:
Builds in main are now failing to build on mono with:
```
 /__w/1/s/src/mono/mono/mini/debugger-agent.c:6406:88: error: too many arguments to function call, expected 6, have 7
          mono_image_load_enc_delta (domain, image, dmeta_bytes, dmeta_len, dil_bytes, dil_len, error);
          ~~~~~~~~~~~~~~~~~~~~~~~~~                                                             ^~~~~
  /__w/1/s/src/mono/mono/mini/../../mono/metadata/metadata-internals.h:905:1: note: 'mono_image_load_enc_delta' declared here
  void
  ^
  1 error generated.
```

Reply:
@stephentoub Fixed in #49099

Reply:
Thanks.